### PR TITLE
Release 202211.0

### PR DESCRIPTION
**Pull request by DmitryLymarenko**
### Overview

Release 202211.0

### CODE REVIEW

File: composer_back.json [hunk 1]
```diff
@@ -0,0 +1,329 @@
+{
```
Comment (limeeugenia) [inline composer_back.json:1]: Please remove file.

File: src/Pyz/Zed/Checkout/CheckoutDependencyProvider.php [hunk 2]
```diff
@@ -48,6 +49,7 @@ protected function getCheckoutPreConditions(Container $container)
     {
         return [
             new CustomerPreConditionCheckerPlugin(),
+            new CustomerAddressSalutationCheckoutPreConditionPlugin(),
```
Comment (limeeugenia) [inline src/Pyz/Zed/Checkout/CheckoutDependencyProvider.php:52]: No `CustomerAddressSalutationCheckoutPreConditionPlugin` here. Is this correct?

Comment (limeeugenia) [reply src/Pyz/Zed/Checkout/CheckoutDependencyProvider.php:52]: Discussed, the diff is related to the guest cart feature which is absent in the b2b. All good here.

**Review by limeeugenia — APPROVED** [review]

File: src/Pyz/Zed/Sales/Presentation/Detail/boxes/customer.twig [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            <div class="col-xs-12 col-md-6">
+                <dl>
+                    <dt>{{ 'Reference' | trans }}</dt>
+                    <dd><a href="{{ url('/customer/view', {'id-customer': order.fkCustomer}) }}" target="_blank">{{ order.customerReference }}</a></dd>
+                    <dt>{{ 'Name' | trans }}</dt>
+                    <dd>{{ order.firstName ~ ' ' ~ order.lastName }}</dd>
+                    <dt>{{ 'Email' | trans }}</dt>
+                    <dd>{{ order.email }}</dd>
+                    <dt>{{ 'Previous orders count' | trans }}</dt>
+                    <dd>{{ order.totalOrderCount }}</dd>
```
Comment (FRITS-Kh) [inline src/Pyz/Zed/Sales/Presentation/Detail/boxes/customer.twig:13]: Shouldn't it be the same as at the core?
```suggestion
                    <dd>{{ order.totalOrderCount | executeFilterIfExists('formatInt', app.locale) }}</dd>
```

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: src/Pyz/Zed/Sales/Presentation/Detail/boxes/customer.twig (hunk 3 above)
Comment (limeeugenia) [reply src/Pyz/Zed/Sales/Presentation/Detail/boxes/customer.twig:13]: This a part December release, not this one.

**Review by FRITS-Kh — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -11,13 +11,13 @@ on:
...
         runs-on: ubuntu-18.04
         strategy:
             fail-fast: false
             matrix:
                 php-version: [
-                        '7.4',
+                        '8.0',
                 ]
 
         env:
```
Comment (FRITS-Kh) [inline .github/workflows/ci.yml:23]: Please update the `node-version` version from `12` to `16` and the `Frontend install-dependencies` according to the [suite](https://github.com/spryker-shop/suite/blob/master/.github/workflows/ci.yml#L117)

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -30,41 +28,35 @@
...
     },
     "devDependencies": {
-        "@babel/plugin-proposal-class-properties": "~7.10.4",
-        "@babel/plugin-transform-runtime": "~7.11.5",
-        "@babel/preset-typescript": "~7.10.4",
+        "@babel/plugin-proposal-class-properties": "~7.17.12",
+        "@babel/plugin-transform-runtime": "~7.17.12",
+        "@babel/preset-typescript": "~7.17.12",
         "@jsdevtools/file-path-filter": "~3.0.2",
         "@spryker/frontend-config.eslint": "~0.0.2",
```
Comment (FRITS-Kh) [inline package.json:52]: devDependencies `"prettier": "~2.5.1"` and `"@prettier/plugin-xml": "~0.13.1"` are missed.

**Review by FRITS-Kh — CHANGES_REQUESTED** [review]

**Review by FRITS-Kh — APPROVED** [review]

**Review by limeeugenia — APPROVED** [review]

**Pull request merged**